webapp/src/ts/modules/messages/messages-content.component.ts

```diff
@@ -101,6 +105,14 @@ export class MessagesContentComponent implements OnInit, OnDestroy, AfterViewIni
       });
   }
 
+  ngAfterViewChecked(): void {
+    const lastMessage = $('.message-content-wrapper .last-message');
+    // Scrolling only when last message is rendered.
+    if (this.hasToScroll && !this.loadingContent && lastMessage.length) {
+      this.scrollToUnread();
```

Comment:
I'm not sure this is triggered correctly. I loaded a conversation with many read messages and was still scrolled all the way to the bottom :(

Reply:
It's hard to determine when the view has finished rendering so the component calculates the scroll correctly. Sadly, I had to add a timeout for now and added a [ToDo for improvement](https://docs.google.com/spreadsheets/d/13BBw8cyV8CLMOzZAR_WDBqkaNrywNOk-i7b7pcypWJ8/edit#gid=303874733)